client/src/components/shared/Info/Notices.js

```diff
@@ -11,14 +12,20 @@ type NoticesProps = {|
 
 const Notices = ({data}: NoticesProps) => (
   <InfoButton
-    label="Zmluvy so štátom"
+    label="Obstarávania"
     count={data.count}
     priceSum={data.total_final_value_amount_eur_sum}
     list={data.most_recent}
     buildItem={(notice: NoticeNew) => (
       <li key={notice.id} title={notice.title}>
```

Comment:
Since `notice.title` is displayed now, is there a reason to also show it on hover?

Reply:
Good point, removing.